JetAnalyzers/bin/jet_correction_analyzer_x.cc

```diff
@@ -603,6 +604,13 @@ int main(int argc,char**argv)
 
          if(nrefmax>0 && JRAEvt->nref>nrefmax) JRAEvt->nref = nrefmax;
          for (unsigned char iref=0;iref<JRAEvt->nref;iref++) {
+	    /*      
```

Comment:
Do not leave unused, commented code. If it's commented because it's only sometimes used, then there should be a switch. If it's commented because you want to reference the old code occasionally you should look at the git history.

Reply:
Okay, I'll remove the commented out code in this case.